cannot find module 'pouchdb/extras/promise'

Did a fresh npm install of this, and it complains as follows:

```
module.js:341
    throw err;
    ^

Error: Cannot find module 'pouchdb/extras/promise'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/Users/aschamp/pace-dx/node_modules/pouchdb-replication-stream/lib/pouch-utils.js:3:15)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

I seem to recall working through this on another pouchdb plugin module, but I can't recall, and for the life of me I cannot find it.  I think the solution was fairly straightforward though.

I installed pouchdb with the 'pouchdb-node' package, since this is just for some commandline utils.

--- a/lib/index.js
+++ b/lib/index.js
@@ -4,7 +4,7 @@ var utils = require('./pouch-utils');
 var version = require('./version');
 var ldj = require('ldjson-stream');
 var through = require('through2').obj;
-var pick = require('lodash/object/pick');
+var pick = require('lodash.pick');
 var toBufferStream = require('./to-buffer-stream');
 var DEFAULT_BATCH_SIZE = 50;
 

--- a/lib/pouch-utils.js
+++ b/lib/pouch-utils.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var Promise = require('pouchdb/extras/promise');
+var Promise = require('pouchdb-promise');
 
 /* istanbul ignore next */
 exports.once = function (fun) {

--- a/lib/writable-stream.js
+++ b/lib/writable-stream.js
@@ -1,7 +1,7 @@
 'use strict';
 
 var utils = require('./pouch-utils');
-var Promise = require('pouchdb/extras/promise');
+var Promise = require('pouchdb-promise');
 var ERROR_REV_CONFLICT = {
   status: 409,
   name: 'conflict',

--- a/package.json
+++ b/package.json
@@ -37,10 +37,10 @@
     "argsarray": "0.0.1",
     "inherits": "~2.0.1",
     "ldjson-stream": "^1.2.1",
-    "lie": "^2.6.0",
-    "lodash": "^3.3.0",
+    "lodash.pick": "^4.0.0",
+    "pouchdb-promise": "^5.4.4",
     "pouch-stream": "^0.4.0",
-    "through2": "^0.6.1"
+    "through2": "^2.0.0"
   },
   "devDependencies": {
     "bluebird": "^1.0.7",
@@ -52,12 +52,13 @@
     "http-server": "~0.8.5",
     "istanbul": "^0.2.7",
     "jshint": "^2.3.0",
+    "lie": "^3.1.0",
     "memorystream": "^0.3.0",
     "mkdirp": "^0.5.0",
     "mocha": "~1.18",
     "noms": "0.0.0",
     "phantomjs": "^1.9.7-5",
-    "pouchdb": "^5.0.0",
+    "pouchdb-memory": "^1.0.0",
     "random-document-stream": "0.0.0",
     "request": "^2.36.0",
     "sauce-connect-launcher": "^0.4.2",

--- a/test/test.js
+++ b/test/test.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var Pouch = require('pouchdb');
+var Pouch = require('pouchdb-memory');
 //var Readable = require('stream').Readable;
 //var Writable = require('stream').Writable;
 

